src/structures/Guild.js

```diff
@@ -884,6 +885,7 @@ class Guild {
     if (typeof data.icon !== 'undefined') _data.icon = data.icon;
     if (data.owner) _data.owner_id = this.client.resolver.resolveUser(data.owner).id;
     if (typeof data.splash !== 'undefined') _data.splash = data.splash;
+    if (typeof data.banner !== 'undefined') _data.banner = data.banner;
```

Comment:
Shouldn't this be resolved to an image too?

Reply:
v11 seems to do this nowhere, just in each respective `set*` method.
However it would be possible to do this, just a bit bothersome without `await`.